Public/Src/Cache/ContentStore/Distributed/NuCache/EventStreaming/ContentLocationEventData.cs

```diff
@@ -203,45 +204,51 @@ public sealed class AddContentLocationEventData : ContentLocationEventData
         /// </summary>
         public IReadOnlyList<long> ContentSizes { get; }
 
+        /// <summary>
+        /// Whether the operation should cuase the content last access time to be updated.
+        /// </summary>
+        public bool ShouldTouch { get; }

```

Comment:
>ShouldTouch [](start = 20, length = 11)

Changing this introduces a breaking change to the serialization protocol for event hub. This will cause errors. You need to make this change in a backward compatible manner.

Reply:
Very well. I'll add a new event, something like "AddWithoutTouching"

Reply:
Very well. I'll add a new event, something like "AddWithoutTouching"

---
In reply to: [341698926](https://github.com/microsoft/BuildXL/pull/1141#discussion_r341698926) [](ancestors = 341698926)